docker/docker-compose-postgres.yml

```diff
@@ -23,6 +23,7 @@ services:
     privileged: true
     volumes:
       - ../:/candlepin-dev
+      - ${SONAR_CRT:-../server/conf/candlepin-redhat-ca.crt}:/candlepin-dev/certs/SONAR.crt
```

Comment:
@Lorquas, Updated the code to access the `SONAR_CRT` as a global environment variable for the cert path. 
Can you add the cert path into the Jenkins env?